crates/tauri-runtime/src/webview.rs

```diff
@@ -444,6 +448,23 @@ impl WebviewAttributes {
     self.background_color = Some(color);
     self
   }
+
+  /// Set whether background throttling should be disabled.
+  ///
+  /// By default, browsers throttle timers and even unload the whole tab (view) to free resources after roughly 5 minutes when
+  /// a view became minimized or hidden. This will permanently suspend all tasks until the documents visibility state
+  /// changes back from hidden to visible by bringing the view back to the foreground.
+  ///
+  /// ## Platform-specific
+  ///
+  /// - **Linux / Windows / Android**: Unsupported yet. But workarounds like a pending WebLock transaction might suffice.
```

Comment:
```suggestion
  /// - **Linux / Windows / Android**: Unsupported. Workarounds like a pending WebLock transaction might suffice.
```

I think there should also be a line for ios / macos mentioning the minimum OS version since they are quite recent.

Needs to be applied to the docs in the other files as well of course.

Reply:
I'll update it tomorrow! Thanks a lot!

Btw: I'm still running my app on this PR and I haven't had a single issue since! I'm certain this will benefit a lot of devs.

Thank you for taking the time to review it @FabianLars